app/src/consts.js

```diff
@@ -4,6 +4,10 @@ export const STAGE_HOME = 0;
 export const STAGE_TRANSCRIBING = 1;
 export const STAGE_TRANSCRIBED = 2;
 export const STAGE_SUMMARIZE = 3;
```

Comment:
I dont think we have a STAGE_SUMMARIZE any more.

Reply:
Should we rename the stages below it too then?